.rubocop.yml

```diff
@@ -0,0 +1,15 @@
+---
+# Managed by modulesync - DO NOT EDIT
+# https://voxpupuli.org/docs/updating-files-managed-with-modulesync/
```

Comment:
Is it?

Reply:
Cut'n'paste. :)  In my defense, I doubt that is the `.rubocop.yml` that is going to be merged.

Reply:
fixed